src/main/java/uk/gov/digital/ho/pttg/AuditService.java

```diff
@@ -36,6 +44,15 @@ public void add(AuditableData auditableData) {
 
     }
 
+    private void checkNamespace(AuditableData auditableData) {
+        String eventNamespace = auditableData.getDeploymentNamespace();
+
+        if (!eventNamespace.equals(auditingDeploymentNamespace)) {
```

Comment:
Can we have an event for this condition please?

Reply:
Done